src/org/zaproxy/zap/extension/importurls/resources/help/contents/importUrls.html

```diff
@@ -7,6 +7,9 @@
 </HEAD>
 <BODY BGCOLOR="#ffffff">
 <H1>Import URLs</H1>
-This add-on adds an option to import a file of URLs. The file must be plain text with one URL per line.<br>
+This add-on adds an option to import a file of URLs. The file must be plain text with one URL per line.
+</p>
+This add-on also exposes a web API endpoint <tt>/importurls/importurls (filePath*)</tt> to facilitate programmatic 
```

Comment:
Worth mentioning "ZAP API"? (Which is easier to find more info on how to work with the endpoint?)